components/dropdown/dropdown-opener-mixin.js

```diff
@@ -204,6 +202,13 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 		this._closeTimerStart();
 	}
 
+	__onDropdownMouseUp() {
+		this._isOpen = true;
+		this._isFading = false;
+		this._isOpenedViaClick = true;
```

Comment:
Setting _isOpenedViaClick will prevent the dropdown from closing on hovering away.